app/services/cleaners/cracked_detail_cleanable.rb

```diff
@@ -0,0 +1,12 @@
+module Cleaners
+  module CrackedDetailCleanable
+    private
+
+    def clear_cracked_details
+      claim.trial_fixed_notice_at = nil if trial_fixed_notice_at
+      claim.trial_fixed_at = nil if trial_fixed_at
+      claim.trial_cracked_at = nil if trial_cracked_at
+      claim.trial_cracked_at_third = nil if trial_cracked_at_third
```

Comment:
**nit** if missing is delegated to claim in the `BaseClaimCleaner` do we need `claim.whatever`?

Reply:
oh you might due to assignment actually but perhaps `self.whatever = `